src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -187,4 +309,16 @@ private function isCommentLastLineToken(Tokens $tokens, $index)
 
         return $content !== ltrim($content, "\r\n");
     }
+
+    /**
+     * Checks if token is new line.
+     *
+     * @param Token $token
+     *
+     * @return bool
+     */
+    private function isNewLine(Token $token)
+    {
+        return strpos(trim($token->getContent(), " \r\t\0\x0B"), "\n") !== false;
```

Comment:
I theory one could use `\r` only, but this very uncommon (https://en.wikipedia.org/wiki/Newline), typically `\n` for Unix and MacOS, `\r\n` for Windows. We don't support `\r` only, so don't worry about it.